core/templates/dev/head/domain/collection/CollectionNodeObjectFactory.js

```diff
@@ -17,20 +17,30 @@
  * collection node domain objects.
  */
 
-// TODO(bhenning): Implement validation functions and related tests.
+// TODO(bhenning): This should be provided by the backend.
+oppia.constant('ACTIVITY_STATUS_PRIVATE', 'private');
+
 oppia.factory('CollectionNodeObjectFactory', [
-      'SkillListObjectFactory', function(SkillListObjectFactory) {
+      'SkillListObjectFactory', 'ACTIVITY_STATUS_PRIVATE',
+      function(SkillListObjectFactory, ACTIVITY_STATUS_PRIVATE) {
+    var _initializeExplorationDetails = function(
+        node, explorationSummaryBackendObject) {
+      node._explorationTitle = explorationSummaryBackendObject.title;
```

Comment:
Agreed, I removed it in favor of grabbing the values from the summary object within the getters themselves.